org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -2056,6 +2123,29 @@ protected String buildDataSQL(int start, int end) {
 		return dataSql;
 	}
 
+    /**
+     * Join tables required for the ORDER BY clause
+     * @param orderClause
+     * @return SQL JOIN
+     */
+    private String getSQLJoin(String sqlMain) {
+		if(Util.isEmpty(joinTableForUserOrder))
+			return "";
+		StringBuilder builder = new StringBuilder();
+		for(TableInfo tableInfo : tableInfos) {
+			if(joinTableForUserOrder.equalsIgnoreCase(tableInfo.getTableName())) {
+				return "";
+			}
+		}
+		if(!sqlMain.contains(" LEFT JOIN " + joinTableForUserOrder)) {
+			builder.append(" LEFT JOIN ").append(joinTableForUserOrder).append(" ON (")
+				.append(joinTableForUserOrder).append(".").append(joinTableForUserOrder).append("_ID = ")
+				.append(getTableName()).append(".").append(joinTableForUserOrder).append("_ID)");
```

Comment:
this needs to be changed to cater for https://idempiere.atlassian.net/browse/IDEMPIERE-5567